src/metabase/driver.clj

```diff
@@ -746,6 +742,19 @@
 
 (defmethod current-db-time ::driver [_ _] nil)
 
+(defmethod db-default-timezone ::driver
```

Comment:
Not 100 million percent related but I updated the sync-timezone code to use `db-default-timezone` instead of `current-db-time` since `current-db-time` is deprecated and this method is its replacement